scripts/verify_meilisearch.py

```
@@ -0,0 +1,44 @@
+    print(f"  Quintile: {hit.get('quintile')}")
+    print(f"  2024 Pass %: {hit.get('percent_achieved_2024')}")
+
+# Stats
+stats_url = "https://search.matrictrend.app/indexes/schools/stats"
+stats_req = urllib.request.Request(
+    stats_url,
+    headers={"Authorization": f"Bearer {MEILI_KEY}"},
+)
+stats_resp = urllib.request.urlopen(stats_req, timeout=15)
+stats = json.loads(stats_resp.read())
+print(f"\nIndex stats:")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)